etl/planning_data/obtain_livestream_data_and_load_into_database.py

```diff
@@ -0,0 +1,310 @@
+import json
+import datetime
+import os
+import requests
+import psycopg2
+import address_data
+
+def main():
+    connection = get_connection()
+    cursor = connection.cursor()
+    cursor.execute("TRUNCATE planning_data")
```

Comment:
Is this meant to be re-run regularly? Could we do something more conservative than `TRUNCATE` and reload all? It seems quite risky to drop everything and hope the bulk reload works okay every time.